astropy/visualization/tests/test_histogram.py

```diff
@@ -49,6 +49,7 @@ def test_hist_specify_ax(rseed=0):
 
 
 @pytest.mark.skipif('not HAS_PLT')
+@pytest.mark.filterwarnings('ignore:p0 does not seem to accurately represent')
```

Comment:
Should we add a note to remove this filter when #9568 is fixed?

Reply:
I'll add a note in #9568

Reply:
Warning now removed, so resolving this.